### ISSUE TITLE
Issue with the questions / solutions for section 19.3 Unquoting

### ISSUE THREAD

**Issue by pieterjanvc** [Tazinho/Advanced-R-Solutions#204]
Hi,

Let me start by thanking you for this great handbook and the exercises! I'm new to the whole TidyEval and this book is really helping me.

While doing the exercises (and looking at solutions) for **section 19.3**, I came across two inconsistencies:

**Question 3**
You expect the following output (given in question)
`(x + y) + (y + z) - (x + y)`

Though I only could get the code to look like this
```
expr(!!xy + !!yz -!!xy)
x + y + (y + z) - (x + y)
```
This is also the solution. However, the questions seems to suggest you need the parenthesis around the first x + y which is confusing and made me spent 15 min trying to achieve just that :)

**Question 8**
Your question asks for the following output
`foo(a = x + y, b = y + z)`

But in your answer, you forgot to use the !! resulting in the output:
`foo(a = xy, b = yz)`

Keep up the great work!
PJ

Find my on the RStudio Community forum as [pieterjanvc](https://community.rstudio.com/u/pieterjanvc/)

**Comment by Charles926** [comment]
Hi all,
I'd like to thank you the same way pieterjanvc did: You really did an amazing job with this book.
To complete the answer, it is possible to obtain the parenthesis by writing double parenthesis around the first unquoting. I don't really understand why it works, but it does.
`expr(((!!xy)) + !!yz - !!xy)`

**Comment by henningsway** [comment]
We are really happy to hear your positive feedback. 😊 

We will take a look at these detailed questions some time soon and get back to you.

Keep enjoying the RRRide! :)

**Comment by henningsway** [comment]
Thank you so much. I just fixed the bug with the missing unquotings!

I agree, it's confusing, that the partheses don't show up in the unquoted expression. As a test I changed the expression to `expr(!!xy * !!yz * !!xz)` which shows up correctly as `(x + y) * (y + z) * (x + z)`. I also don't fully understand the (AST based) mechanism behind this. But the correct computations seem to be ensured. :)

Thank you for your attention to detail!
